plugins/itemencrypted/itemencrypted.cpp

```diff
@@ -132,6 +132,25 @@ const QString &gpgExecutable()
     return gpg;
 }
 
+struct KeyPairPaths {
+    KeyPairPaths()
+    {
+        const QString path = getConfigurationFilePath("");
+        sec = QDir::toNativeSeparators(path + ".sec");
+        pub = QDir::toNativeSeparators(path + ".pub");
+
+#ifdef Q_OS_WIN
+        if (checkUnixGpg(gpgExecutable())) {
+            pub.replace('\\', '/').replace(":", "").insert(0, '/');
+            sec.replace('\\', '/').replace(":", "").insert(0, '/');
```

Comment:
How about using `QDir::fromNativeSeparators` instead of replacing the chars?

```suggestion
            pub = QDir::fromNativeSeparators(pub).replace(":", "").insert(0, '/');
            sec = QDir::fromNativeSeparators(sec).replace(":", "").insert(0, '/');
```

Or even better skip the previous `QDir::toNativeSeparators()` calls in this case.

Reply:
Seems to work. Thanks